src/app/caesar-portal/group-area/group-area.component.ts

```diff
@@ -37,10 +40,34 @@ export class GroupAreaComponent implements OnInit, OnDestroy {
     }
 
     private getGroup (groupId: number) {
-        this.groupService.get(groupId).then(
+        this.groupService.get(groupId).subscribe(
             (group: Group) => {
                 this.group = group;
-                this.groupService.setGroup(group);
+                this.setGroupCurrent(this.group);
+                this.getLocation(this.group._links['location'].href);
+                this.getStage(this.group._links['status'].href);
+            },
+            error => console.log(error)
+        );
+    }
+
+    setGroupCurrent(group: Group): void {
+        this.groupService.setGroupCurrent(group);
+    }
+
+    private getLocation (locationUrl: string) {
+        this.groupService.getLocation(locationUrl).subscribe(
+            (location: Location) => {
+                this.location = location;
+            },
+            error => console.log(error)
+        );
+    }
+
+    private getStage (stageUrl: string) {
+        this.groupService.getParametr(stageUrl).subscribe(
+            (stage: string[]) => {
+                this.stage = stage;
```

Comment:
stage is an array of strings? if so maybe it is reasonable to rename it to stages? otherwise fix type

Reply:
yes, stage is an array of strings, but not stages, it's an array of stage's properties.